src/Pact/Types/Time.hs

```diff
@@ -0,0 +1,497 @@
+{-# LANGUAGE CPP #-}
```

Comment:
Let's get rid of the CPP and just pin to supported `time` versions

Reply:
I tried that. Unfortunately, I wasn't able to resolve the dependencies of the nix builds when up-pinning time to 1.9.